docs/quickstart.md

```diff
@@ -44,17 +44,16 @@ You can create a script with the following content to run fractal-web installed
 export FRACTAL_SERVER_HOST=http://localhost:8000
 export PUBLIC_FRACTAL_ADMIN_SUPPORT_EMAIL=help@localhost
 export PUBLIC_OAUTH_CLIENT_NAME=
-# remember to set this in production
+# remember to set this in production (e.g. subdomain.example.org)
 export AUTH_COOKIE_DOMAIN=
-# set this to true in production
-export AUTH_COOKIE_SECURE=false
```

Comment:
to verify

Reply:
I confirm that `Secure=true` works on localhost.

Reply:
Great, thanks. Then let's not include it, in order to keep it brief